config/routes.rb

```diff
@@ -145,7 +151,7 @@ def scoped_navigation_routes(context, navigation, as_redirects: false)
     ### END Hub Admin routes (Case Management)
 
     # Any other top level slash just goes to home as a source parameter
-    get "/:source" => "public_pages#home", constraints: { source: /[0-9a-zA-Z_-]{1,100}/ }
+    get "/:source" => "public_pages#redirect_locale_home", constraints: { source: /[0-9a-zA-Z_-]{1,100}/ }
```

Comment:
this accomplishes 2 things -- it ensures we are consistent in including locales, and the redirect also fixes duplicate content issues that this approach causes e.g. /211 and / and /en and /whatevs all have the same content and really should be treated as a single canonical page for search. and now they are, because they all redirect to /en